std/process.d

```diff
@@ -1081,6 +1081,17 @@ version (Posix) @system unittest
     std.file.write(directory, "foo");
     scope(exit) remove(directory);
     assertThrown!ProcessException(spawnProcess([prog.path], null, Config.none, directory));
+
+    // can't run in directory if user does not have search permission on this directory
```

Comment:
I'm not sure here. What does "search permission" mean then in chdir documentation?
If it's read permission, then there should be no such test, since it relies on non-standard behavior.